nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -50,14 +53,7 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.text.ParseException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
This guy is still giving checkstyle errors. To run this locally you can use the contrib-check profile when building. I usually do this from the lowest module I'm changing, so nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors:

mvn clean install -Pcontrib-check